src/go/rpk/pkg/cli/cluster/quotas/import.go

```diff
@@ -0,0 +1,275 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package quotas
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/kafka"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/franz-go/pkg/kadm"
+	"github.com/twmb/types"
+	"gopkg.in/yaml.v3"
+)
+
+// quotasDiff represents a delta in the quotas after importing a quota source.
+type quotasDiff struct {
+	EntityStr string `json:"entity,omitempty" yaml:"entity,omitempty"`
+	QuotaType string `json:"quota-type,omitempty" yaml:"quota-type,omitempty"`
+	OldValue  string `json:"old-value,omitempty" yaml:"old-value,omitempty"`
+	NewValue  string `json:"new-value,omitempty" yaml:"new-value,omitempty"`
+}
+
+func importCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var (
+		source    string
+		noConfirm bool
+	)
+	cmd := &cobra.Command{
+		Use:   "import",
+		Short: "Import client quotas",
+		Long: `Import client quotas.
+
+You can use this command to import client quotas in the format produced by 
+'rpk cluster quotas describe --format json/yaml'. To view the schema of the 
+import string, use 'rpk cluster quotas describe --format help'.
```

Comment:
Do we need to pass the flag to get this specific help text?

Reply:
Yes, for the schema (and pretty much is the same for every command that supports the `--format` flag)

`rpk command --format help` will show the expected return schema.

However, I added the schema to the help text :+1:

Reply:
huh I didn't know that. Maybe we can add something on website docs to talk about it